toolkits/global/packages/global-ad/scss/50-components/ad.scss

```diff
@@ -48,9 +46,17 @@
 	}
 }
 
+.js .c-ad--160x600 {
+	@include media-query('md') {
+		display: block;
+	}
+}
+
 .c-ad--300x250 {
 	// MPU
+	display: none;
```

Comment:
Could use the max media query here to avoid the override later?

Reply:
needed the override with `.js` otherwise empty space when no js. Most noticeable with MPU on narrow screen width.